modules/workflows/ModalityGrouping.py

```diff
@@ -15,14 +15,15 @@ def modality_split(cold_extraction_path, modality_split_path):
                 dcm_file = pyd.dcmread(dcm_filename)
                 dcm_modality = dcm_file.Modality
 
-                print (dcm_modality, dcm_only_folder)
+                # print (dcm_modality, dcm_only_folder)
                 isExist = os.path.exists(modality_split_path+str(dcm_modality)+'/'+str(dcm_only_folder))
                 if not isExist:
                     os.makedirs(modality_split_path+str(dcm_modality)+'/'+str(dcm_only_folder))
-                    shutil.copy2(src=cold_extraction_path+str(dcm_path), dst=modality_split_path+str(dcm_modality)+'/'+str(dcm_only_folder)+'/'+str(dcm_path.split('/')[-1]))
+                print (cold_extraction_path+str(dcm_path), modality_split_path+str(dcm_modality)+'/'+str(dcm_only_folder)+'/'+str(dcm_path.split('/')[-1]))
+                shutil.copy2(src=cold_extraction_path+str(dcm_path), dst=modality_split_path+str(dcm_modality)+'/'+str(dcm_only_folder)+'/'+str(dcm_path.split('/')[-1]))
 
 if __name__ == "__main__":
-    cold_extraction_path = sys.argv[1]
-    modality_split_path = sys.argv[2]
+    cold_extraction_path = '/home/aredd30/larynx_extract/cold_extraction/' # sys.argv[1]
```

Comment:
Why are you introducing hard-coded values here?

Reply:
Just forgot to remove them after validating the changes. I will remove them and submit the PR again.